scrycall/scry_args.py

```diff
@@ -0,0 +1,68 @@
+import argparse
+from .scry_output import Attributes
+
+
+class ArgumentError(Exception):
+    """Error with input arguments"""
+
+
+def parse_args(args: list) -> argparse.Namespace:
+    """Parse command line arguments
+
+    Args:
+        args: sys argv
+
+    Returns:
+        parsed_args: Args namespace
+    """
```

Comment:
This header comment is taking up a lot of space. I don't see any reason this couldn't just be one line. I might even argue that there be no comment at all, as the name of the function is fairly self-explanatory.

Reply:
In public facing code (or really any code in general) having function/method comments are standard practice and can be helpful for others when reading your code and for tooltips in IDEs when browsing from other files.

This style used here is Google's but i'll leave it up to you if you want to reject.